setup.py

```diff
@@ -17,11 +18,30 @@
     license="MIT",
     version='0.1.0',
     install_requires=[
-        # "catalystcoop.pudl @ git+https://github.com/catalyst-cooperative/pudl.git@dev",
+        "catalystcoop.pudl @ git+https://github.com/catalyst-cooperative/pudl.git@mul_eia",
```

Comment:
Will this break if we merge in `mul_eia` into dev and delete the branch?

Reply:
Not if but WHEN!

Yes this is janked AF but until we get all this stuff into released versions I don't know what else to do.